monitoring/roles/loki/files/values.yaml

```diff
@@ -38,7 +38,7 @@ loki:
     enabled: true
     storageClassName: do-block-storage
     accessModes: ["ReadWriteOnce"]
-    size: 50Gi
+    size: 100Gi
```

Comment:
Were you able to get some metrics so far? How much space do logs take up per day? And is there some clean up policy or log retention policy?

Reply:
Hard to say per day the exact amount since it depends on what's generating logs. I used the pv dashboard on prometheus. Logs are kept for 30 days by default, if we need to we can set up other types of storage for long term storage

Reply:
Loki is already at 25GB in dev, though I checked and there's a ton of debug logs coming in from postgres
https://dev-k8s.treetracker.org/grafana/d/919b92a8e8041bd567af9edab12c840c/kubernetes-persistent-volumes?orgId=1&refresh=10s&var-datasource=default&var-cluster=&var-namespace=monitoring&var-volume=storage-loki-0&from=now-30d&to=now
![image](https://user-images.githubusercontent.com/16839335/168189670-1c848a17-9d41-4d43-828d-01deeb721ac8.png)